app/models/alchemy/element.rb

```diff
@@ -302,8 +302,7 @@ def copy_nested_elements_to(target_element)
       nested_elements.map do |nested_element|
         Element.copy(nested_element, {
           parent_element_id: target_element.id,
-          page_id: target_element.page_id,
-          cell_id: target_element.cell_id
```

Comment:
I don't think we need to remove the `cell_id` here. The actual cause of the issue is the now fixed relation scope. Could you please try without removing it?